patches/chrome-browser-resources-settings-appearance_page-appearance_page.html.patch

```diff
@@ -20,7 +20,7 @@ index bdbf90e2482d817ca0cfddb1fe5b0d7c1de58258..9bb5d2a70a62848821d5a2189ad0de09
          focus-config="[[focusConfig_]]">
        <div route-path="default">
 +<if expr="not _google_chrome">
-+        <settings-brave-appearance-theme prefs="{{prefs}}"></settings-brave-appearance-theme>
++        <settings-brave-appearance-top prefs="{{prefs}}"></settings-brave-appearance-top>
```

Comment:
I don't think the location (top) is relevant to the name

Reply:
Reverted to theme.